wsl-open.sh

```diff
@@ -189,32 +160,12 @@ File=$1
 if [[ -n $File ]]; then
   if [[ -e $File ]]; then
     # File or directory
-    FilePath="$(readlink -f "$File")"
-
-    # shellcheck disable=SC2053
-    if [[ $FilePath != $WslDisks/* ]]; then
```

Comment:
Instead of removing this, add a case before this if statement that checks if `wslpath` is available, and if so, do `FileWin=$(wslpath -w "$FilePath"), otherwise do as normal.